### PR TITLE
Databases-week1

### PR DESCRIPTION
**Pull request by somayehdvr**

### CODE REVIEW

File: databases/week1/exercise.sql [hunk 1]
```diff
@@ -0,0 +1,42 @@
+USE hyf_lesson1;
+
+-- Select the names and phones of all users
+Select name, phone FROM hyf_lesson1.user;
```
Comment (lawansubba) [inline databases/week1/exercise.sql:4]: please have a look at the homework section at this page https://github.com/HackYourFuture-CPH/databases/tree/main/lesson1

File: databases/week1/homework.sql [hunk 2]
```diff
@@ -0,0 +1,53 @@
+-- Find out how many tasks are in the task table
+SELECT COUNT(*)
```
Comment (lawansubba) [inline databases/week1/homework.sql]: COUNT(*) as task_amount
_Pointer_
Provide a column name for columns you have created

File: databases/week1/homework.sql [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+FROM task;
+
+-- Find out how many tasks in the task table do not have a valid due date
+SELECT COUNT(*)
```
Comment (lawansubba) [inline databases/week1/homework.sql]: COUNT(*) as task_count
Pointer
Provide a column name for columns you have created

File: databases/week1/homework.sql [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+FROM task
+WHERE due_date IS NULL;
+
+-- Find all the tasks that are marked as done
+SELECT * 
+FROM task
+JOIN status ON task.status_id = status.id
+WHERE status.name LIKE "%Done%";
```
Comment (lawansubba) [inline databases/week1/homework.sql]: status.name = "Done";
_Pointer_
Use LIKE for pattern matches and perform an equality operation

File: databases/week1/homework.sql [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+
+-- Find all the tasks that are not marked as done
+SELECT * 
+FROM task
+JOIN status ON task.status_id = status.id
+WHERE status.name NOT LIKE "%Done%";
```
Comment (lawansubba) [inline databases/week1/homework.sql]: status.name != "Done";
_Pointer_
Use LIKE for pattern matches and perform an equality operation

File: databases/week1/homework.sql [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+FROM task
+ORDER BY created DESC LIMIT 1;
+
+-- Get the title and due date of all tasks where the title or description contains database
+SELECT title, due_date
+FROM task
+WHERE title LIKE "%database%" OR description LIKE "%database%";
+
+-- Get the title and status (as text) of all tasks
+SELECT title, status.name
```
Comment (lawansubba) [inline databases/week1/homework.sql]: task.title, status.name
_Pointer_
Use tablename.columnname when joins are involved

File: databases/week1/homework.sql [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+FROM task
+JOIN status ON task.status_id = status.id;
+
+-- Get the name of each status, along with a count of how many tasks have that status
+SELECT name, COUNT(*)
```
Comment (lawansubba) [inline databases/week1/homework.sql]: SELECT task.name, COUNT(*) as task_count
_Pointer_
Provide name for columns you have created and use tablename.column when joins are invovled

File: databases/week1/homework.sql [hunk 8]
```diff
@@ -0,0 +1,53 @@
...
+FROM status
+JOIN task ON task.status_id = status.id
+GROUP BY name;
+
+-- Get the names of all statuses, sorted by the status with most tasks first
+SELECT name, COUNT(*)
```
Comment (lawansubba) [inline databases/week1/homework.sql]: SELECT task.name, COUNT(*) as task_count
Pointer
Provide name for columns you have created and use tablename.column when joins are invovled

**Review by lawansubba — COMMENTED** [review]
Well done!! 
Please fix minor comments

**Comment by somayehdvr** [comment]
> Well done!! Please fix minor comments

thanks :)

**Pull request merged**